friends/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import "./App.css";
+
+import FriendsList from "./Components/FriendsList";
+import FriendsForm from "./Components/FriendsForm";
+
+import axios from "axios";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      name: "",
+      age: "",
+      email: "" //placeholder for new friends added
+    };
+  }
+
+  updateFriends = (newFriendsList) => {
+    this.setState({ friends: newFriendsList });
+  };
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then((response) => {
+        this.setState({ friends: response.data });
+      })
+      .catch((err) => console.log(err));
+  }
+
+  onChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  addNewFriend = () => {
```

Comment:
if you add an `event.preventDefault()` it'll stop your page from refreshing when you add a friend and make for a better user experience.